src/frontend/web/src/ratingpage/EffortBar.css

```diff
@@ -0,0 +1,23 @@
+.bar {
+  text-align: center;
+}
+
+.full-bar-element {
+  display: inline-block;
+  width: 1.7vh;
+  height: 1.7vh;
+  background-color: #86b362;
```

Comment:
Please export this color to a constant (in a global scss file) - we may want to use it in other places as part of the theme.

Reply:
Good idea. I've put it inside a new 'assets' folder (in the same hierarchy of 'components', 'ratingPage' and 'homePage' directories)